homeassistant/components/sensor/google_home_alarm.py

```diff
@@ -0,0 +1,146 @@
+"""
+Sensor for retrieving latest alarm from Google Home.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.google_home_alarm/
+"""
+from datetime import timedelta
+
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+from homeassistant.helpers.entity import Entity
+from homeassistant.const import (CONF_HOST, CONF_MONITORED_CONDITIONS,
+                                 CONF_DISPLAY_OPTIONS)
+import homeassistant.util.dt as dt_util
+
+REQUIREMENTS = ['ghlocalapi==0.1.0']
+
+TIME_STR_FORMAT = '%H:%M'
+
+CONF_SHOW_TIMERS = 'show_timers'
+
+ICON = 'mdi:alarm'
+
+SENSOR_TYPE_TIMER = 'timer'
+SENSOR_TYPE_ALARM = 'alarm'
+
+SENSOR_TYPES = {
+    SENSOR_TYPE_TIMER: "Timer",
+    SENSOR_TYPE_ALARM: "Alarm",
+}
+
+DISPLAY_TYPES = {
+    'time': 'Time',
+    'date': 'Date',
+    'date_time': 'Date & Time',
+    'time_date': 'Time & Date',
+    'beat': 'Internet Time',
+    'time_utc': 'Time (UTC)',
+}
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_HOST): cv.string,
+    vol.Optional(CONF_SHOW_TIMERS, default=True): cv.boolean,
+    vol.Optional(CONF_MONITORED_CONDITIONS, default=['alarm', 'timer']):
+        vol.All(cv.ensure_list, [vol.In(SENSOR_TYPES)]),
+    vol.Optional(CONF_DISPLAY_OPTIONS, default='time'): vol.In(DISPLAY_TYPES)
```

Comment:
display formats are a frontend concern. The backend should only deal with iso formatted timestamps

Reply:
will address that as soon as I get home!

Reply:
pushed the code now 👍

Reply:
The Travis build passed now